### PR TITLE
Addeded Comments and Docstrings to Functions in the PhasePlot2d and PhasePlot3d files

### PR DESCRIPTION
**Pull request by SasiMitraB**
The following functions have been slightly modified
1. PhasePlot2d._create_arrays(self),  
2. PhasePlot3d._create_arrays(self), 
3. PhasePlot2d._PolarTransformation(self), 
4. PhasePlot3d._PolarTransformation(self)

The following changes have been made:
* Comments have been added. 
* A detailed docstring explaining the method and the formulae used has been added
* A couple of the lines have been re-written for clarity

### CODE REVIEW

**Comment by SasiMitraB** [comment]
Hey! So I came across this library when looking to do some stuff for my own Non-Linear Dynamics course. I thought it would be a bit cool to add a couple of features, but first I wanna get a hang of the library by making comments and documenting the code.

**Review by QuanticPony — COMMENTED** [review]
So nice to see someone new!!
I hope you liked the library and found it useful. Really appreciate your contribution.
Unfortunately I can't check if the changes break anything today... On a fast lookup it shouldn't, but I will test it nonetheless. Probably tomorrow morning

File: phaseportrait/PhasePortrait3D.py [hunk 1]
```diff
@@ -21,6 +21,41 @@ class PhasePortrait3D:
     -------
     ![image](../../imgs/doc_examples/pp3d_example.png)
 
+    ```python3
```
Comment (QuanticPony) [inline phaseportrait/PhasePortrait3D.py:24]: Does your editor does highlighting with python3? Mine does not. But it works with `py` or `python`. Could you change it to `py` so it's like PhasePotrait2D example?

**Review by QuanticPony — CHANGES_REQUESTED** [review]
Just one more comment. Next time request merge to develop please.
We don't want to update master between releases.
Thank you again for your contribution!

**Comment by QuanticPony** [comment]
Hi @SasiMitraB!
Just to let you know, as it seems that this pull request has been a bit abandoned, I will add the changes in a separate commit in a couple of days with your username as a co-author.
Unless you say otherwise, or fix those minor things, of course
Thanks again for your contribution!

**Comment by QuanticPony** [comment]
Closing this PR as changes have been added on commit https://github.com/phaseportrait/phaseportrait/commit/066f9bd34558e275ed19f1542e4885e142912d21. Thank you again for your contribution!

**Pull request closed without merging**